unstable/wlr-virtual-pointer-unstable-v1.xml

```diff
@@ -29,6 +29,13 @@
       requests are mostly mirror opposites of those specified in wl_pointer.
     </description>
 
+    <enum name="error">
```

Comment:
So the wlroots version and the upstream wlr-protocols version were out-of-sync, right?

Reply:
That appears to be the case. I should put the realignment into a separate commit.